qualtran/bloqs/rotations/phase_gradient_test.py

```diff
@@ -136,36 +180,51 @@ def test_add_into_phase_grad_controlled(controlled: int):
                 )
                 basis_map[input_state] = output_state
                 expected_unitary[output_state, input_state] = 1
+
     # Test cirq style simulation.
     assert len(basis_map) == len(set(basis_map.values()))
     circuit = cirq.Circuit(bloq.on(*cirq.LineQubit.range(num_bits)))
     np.testing.assert_allclose(circuit.unitary(), expected_unitary, atol=1e-8)
 
 
+_ADD_SCALED_VAL_INTO_PHASE_REG_EXAMPLES: list[AddScaledValIntoPhaseReg] = [
+    AddScaledValIntoPhaseReg.from_bitsize(4, 7, 0.123, 6),
+    AddScaledValIntoPhaseReg.from_bitsize(2, 8, 1.3868682, 8),
+    AddScaledValIntoPhaseReg.from_bitsize(4, 9, -9.0949456, 5),
+    AddScaledValIntoPhaseReg.from_bitsize(6, 4, 2.5, 2),
```

Comment:
Maybe try adding a test for `AddScaledValIntoPhaseReg.from_bitsize(8, 3, 7.5, 1)` and see if that fails the classical simulation test due to `x.like()` truncation above?